portfolio/src/main/webapp/script.js

```diff
@@ -76,8 +77,43 @@ const fetchMessageUsingJSON = () => {
     // of messagesObj
     const messagesSize = messagesObj.messages.length;
     const message = messagesObj.messages[Math.floor(Math.random() * messagesSize)];
-
-    // Add message to the page
+    
+    // Add message to the page.
     factContainer.innerText = message;
   });
 };
+
+const showUserComments = () => {
+  fetch('/data').then(response => response.json()).then(comments => {
+    // messagesObj is an object, not a string, so we have to
+    // reference its fields to create HTML content
+    
+    const commentsSize = comments.length;
+
+    // If there are no comments
+    if (commentsSize === 0)
+    {
+      return;
```

Comment:
if-returns don't need an else clause. If we didn't return, we're already in the 'else' case